added warning on top for security reasons (#1969)
spryker/spryker-docs: docs/cloud/dev/spryker-cloud-commerce-os/configuring-debugging.md

  - /docs/cloud/dev/spryker-cloud-commerce-os/debugging
---

{% info_block warningBox "Security" %}

> Debugging mode should be turned off once debugging is completed. Leaving it permanently enabled can lead to disclosure of sensitive information.

Reviewer: ```suggestion
Once debugging is completed, turn off debugging mode. Leaving it permanently enabled can lead to the disclosure of sensitive information.
```
